src/main/scala/net/psforever/objects/serverobject/llu/CaptureFlagSocketControl.scala

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2020 PSForever
+package net.psforever.objects.serverobject.llu
+
+import akka.actor.Actor
+import net.psforever.objects.serverobject.affinity.FactionAffinityBehavior
+
+/**
+  * An `Actor` that handles messages being dispatched to a lattice logic unit (LLU) socket.
+  * Actually does nothing lol
```

Comment:
> All amenities in a facility are to receive those messages in case they are powered. In fact, all objects of type `PlanetSideServerObject` should be expected to have a (working?) control agency

It's probably worth adding something along those lines as a comment to this, so in the future we know why it exists but does nothing

Alternatively, can we just skip over `PlanetSideServerObject` that have a null `Actor`?

Reply:
Technically, it has the `Default.Actor` that bounces messages sent to it into `deadLeaves`, not `null`.  If it were `null`, that part of the server would explode whenever a message was dispatched to it.  The purpose of `Default.Actor` is to avoid that explosion and to indicate that something's actor was not started correctly.

In the documentation for `PlanetSideServerObject`:
"An object layered on top of the standard game object class that maintains an internal `ActorRef`.
"A measure of synchronization can be managed using this `Actor` as a 'controlling agent'."
I can add a comment that "this actor only exists to comply with the contract for `PlanetSideServerObject` entities" but that seems more excessive that the informal "lol".